doc/backend_vars.asciidoc

```diff
@@ -312,6 +312,7 @@ GENERAL_HW_VNC_DEPTH;integer;16;Color depth for VNC server
 GENERAL_HW_VNC_JPEG;integer;0;Advertise support for Tight JPEG encoding
 GENERAL_HW_NO_SERIAL;boolean;;Don't use serial
 GENERAL_HW_VIDEO_STREAM_URL;string;;Video stream URL (in ffmpeg's syntax) to receive, for example 'udp://@:5004' or '/dev/video0'. Using 'ustreamer:///dev/videoN' will use ustreamer from PiKVM instead of ffmpeg to read '/dev/videoN'. Ustreamer support requires pack("D") working, which rules out openSUSE 15.5's perl.
+GENERAL_HW_VIDEO_CMD_PREFIX;string;;Prefix to prepend to 'ffmpeg' and 'v4l2-ctl' commands, can be used to run them on a different host via SSH. Note: the value is tokenized on spaces, so avoid their use.
```

Comment:
Why avoid their use? also, please add an example in the doc.

Reply:
Clarified and added an example.  I meant that a space can cut a parameter in two.